testing.sagebrush.master/Stepwat_Inputs/Input/sxw/Input/siteparam.in

```diff
@@ -44,10 +44,12 @@
 1.1          # range
 
 # ---- Intrinsic site params: Blak, MT (47.06581° N, -108.5826° E,  m elevation) ----
-0.8214533	# latitude of the site in radians
+-0.0000	# longitude of the site in degrees (W < 0; E > 0)
```

Comment:
update to  -108.5826

Reply:
I will do that in a commit 21c8f1d877dc7b25f28536cf1e75337d3d4bef53. Sorry, I didn't realize that you nicely documented which location the values represent!

Reply:
no worries and thanks!